tgarchive/__init__.py

```diff
@@ -82,6 +82,8 @@ def main():
                    dest="template", help="path to the template file")
     b.add_argument("-o", "--output", action="store", type=str, default="site",
                    dest="output", help="path to the output directory")
+    b.add_argument("--symlink", action="store_true", dest="symlink",
+                   help="use symlinks to save disk space")
```

Comment:
Could you please change this message to `symlink media and other static files instead of copying`